api/recipe/v1/recipe.py

```diff
@@ -67,3 +69,65 @@ async def create_recipe(request: UserCreateRecipeRequestSchema):
         CreatorCreateRecipeRequestSchema(creator_id=request.user_id, **request.dict())
     )
     return await RecipeService().get_recipe_by_id(recipe_id)
+
+
+def form_to_schema(
+    name: Annotated[str, Form()],
+    description: Annotated[str, Form()],
+    preparing_time: Annotated[int, Form()],
+    image: Annotated[str, Form()],
+    user_id: Annotated[int, Form()],
+    tags: Annotated[List[int], Form()],
+    instructions: Annotated[List[str], Form()],
+    ingredients: Annotated[List[Dict[str, Union[int, float, str]]], Form()],
+) -> UserCreateRecipeRequestSchema:
+    return UserCreateRecipeRequestSchema(
+        name=name,
+        description=description,
+        preparing_time=preparing_time,
+        image=image,
+        user_id=user_id,
+        tags=tags,
+        instructions=instructions,
+        ingredients=ingredients,
+    )
+
+
+@recipe_v1_router.post(
+    "/test",
+    responses={"400": {"model": ExceptionResponseSchema}},
+    # dependencies=[Depends(PermissionDependency([IsAdmin]))],
+)
+@version(1)
+async def endpoint(
```

Comment:
remove unused function
rename endpoint function and path to something more descriptive